portal-api/src/scheduler/jobs/send-reporting-period-start-emails-job/index.js

```diff
@@ -0,0 +1,69 @@
+const api = require('../../../v1/api');
+const sendEmail = require('../../../external-api/send-email');
+const EMAIL_TEMPLATE_IDS = require('../../../constants/email-template-ids');
+const { hasValue, isValidEmail } = require('../../../utils/string');
+const { getReportPeriod, getReportDueDate } = require('./helpers');
+
+const FIRST_OF_MONTH_AT_7_AM_CRON = '0 7 1 * *';
+const SCHEDULE = process.env.PORTAL_REPORTING_PERIOD_START_EMAIL_SCHEDULE ?? FIRST_OF_MONTH_AT_7_AM_CRON;
+
+const DEFAULT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH = 10;
+const DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH =
+  process.env.PORTAL_REPORTING_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH ?? DEFAULT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH;
+
+const EMAIL_NAME = 'GEF utilisation reporting period start';
+
+const sendEmailForBank = async (bank, reportPeriod, reportDueDate) => {
+  const { name: bankName, paymentOfficerTeam } = bank;
+  const paymentOfficerTeamEmail = paymentOfficerTeam?.email;
+
+  try {
+    if (!hasValue(paymentOfficerTeamEmail)) {
+      console.info(`Not sending ${EMAIL_NAME} email to '${bankName}' - no payment officer team email set`);
```

Comment:
Not that it makes much different in our current logging setup but can this be a console.warn? It'll be shouting a little in private beta but in public beta I'd be expecting us to opt everyone in quickly and onboard banks with this asap